src/tcgwars/logic/impl/gen4/Stormfront.groovy

```diff
@@ -119,11 +119,11 @@ public enum Stormfront implements LogicCardInfo {
   SWITCH_93 ("Switch", "93", Rarity.UNCOMMON, [TRAINER]),
   CYCLONE_ENERGY_94 ("Cyclone Energy", "94", Rarity.UNCOMMON, [SPECIAL_ENERGY, ENERGY]),
   WARP_ENERGY_95 ("Warp Energy", "95", Rarity.UNCOMMON, [SPECIAL_ENERGY, ENERGY]),
-  DUSKNOIR_LV_X_96 ("Dusknoir LV.X", "96", Rarity.HOLORARE, [LEVEL_UP, EVOLUTION, POKEMON, _PSYCHIC_]),
-  HEATRAN_LV_X_97 ("Heatran LV.X", "97", Rarity.HOLORARE, [LEVEL_UP, EVOLUTION, POKEMON, _FIRE_]),
-  MACHAMP_LV_X_98 ("Machamp LV.X", "98", Rarity.HOLORARE, [LEVEL_UP, EVOLUTION, POKEMON, _FIGHTING_]),
-  RAICHU_LV_X_99 ("Raichu LV.X", "99", Rarity.HOLORARE, [LEVEL_UP, EVOLUTION, POKEMON, _LIGHTNING_]),
-  REGIGIGAS_LV_X_100 ("Regigigas LV.X", "100", Rarity.HOLORARE, [LEVEL_UP, EVOLUTION, POKEMON, _COLORLESS_]),
+  DUSKNOIR_LV_X_96 ("Dusknoir Lv.X", "96", Rarity.HOLORARE, [LVL_X, POKEMON, _PSYCHIC_]),
+  HEATRAN_LV_X_97 ("Heatran Lv.X", "97", Rarity.HOLORARE, [LVL_X, POKEMON, _FIRE_]),
+  MACHAMP_LV_X_98 ("Machamp Lv.X", "98", Rarity.HOLORARE, [LVL_X, POKEMON, _FIGHTING_]),
+  RAICHU_LV_X_99 ("Raichu Lv.X", "99", Rarity.HOLORARE, [LVL_X, POKEMON, _LIGHTNING_]),
+  REGIGIGAS_LV_X_100 ("Regigigas Lv.X", "100", Rarity.HOLORARE, [LVL_X, POKEMON, _COLORLESS_]),
```

Comment:
The implementations 👇 needs some ❤ as well

Reply:
I'm planning to have the implementations be in a separate PR